blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Dasharo compatible with MSI PRO Z690-A release v1.1.1
+abstract: 'A short overview of changes introduced in v1.1.1
+           release of Dasharo compatible with MSI PRO Z690-A
+	   with a short demo of Qubes OS 4.1.2.'
```

Comment:
This post should be backlincked in release notes.

Reply:
That's the goal